app/src/layouts/tabular/tabular.vue

```diff
@@ -1,5 +1,5 @@
 <template>
-	<div class="layout-tabular">
+	<div :class="$attrs.class" class="layout-tabular">
```

Comment:
let's make sure to add this to other layouts as well 👍 (so the Files module with Cards layout can also benefit from this for example)

Reply:
I don't like this approach as this is unintuitive and easy to miss when writing new layouts.
I'd rather apply the `--layout-offset-top` not to the element with the layout class but the whole private view like so:
```css
.private-view {
    --layout-offset-top: calc(var(--header-bar-height) - 1px);
}
```

That way you always have access to `--layout-offset-top` no matter if you do `:class="$attrs.class"` or not. Also don't forget to remove `class="layout"` from the component as it has no purpose.

Reply:
> I don't like this approach as this is unintuitive and easy to miss when writing new layouts.

This is definitely something I'm worried about in my latest review! Whole hearted agree with your solution here 👍